contract/src/api.rs

```diff
@@ -172,6 +173,7 @@ impl Contract {
         if attached_balance > 0 {
             env::panic_str("The attached deposit could not be added to any account");
         }
+        return SetOutput { block_height: U64(near_sdk::env::block_height()) };
```

Comment:
In Rust, the last statement is the return value, so you don't need `return` here:

```suggestion
        SetOutput { block_height: U64(near_sdk::env::block_height()) }
```